frontend/src/scenes/web-analytics/WebAnalyticsHeaderButtons.tsx

```diff
@@ -0,0 +1,98 @@
+                                <h3 className="font-semibold flex items-center gap-2">
+                                    About the New Query Engine
+                                    <LemonTag type="warning" className="uppercase">
+                                        Beta
+                                    </LemonTag>
+                                </h3>
+                            </div>
+                            <p className="mb-3">
+                                Our new Web Analytics Query Engine powers faster queries using pre-aggregated data,
+                                giving you quicker access to insights and it's much better at handling large datasets.
+                            </p>
+
+                            <div className="mb-3">
+                                <strong>A few things to note:</strong>
+                                <ul className="list-disc ml-4 mt-1 space-y-1">
+                                    <li>
+                                        Some filters may not yet be supported, but we're working on expanding coverage.
+                                    </li>
+                                    <li>
+                                        We use smart approximation techniques to keep performance high, and we aim for
+                                        less than 1% difference compared to exact results.
+                                    </li>
+                                    <li>
+                                        You can toggle the engine on or off directly from this interface if you want to
+                                        compare results or prefer the previous method.
+                                    </li>
+                                </ul>
+                            </div>
+
+                            <div className="mb-3">
+                                <strong>Coming Soon:</strong>
+                                <ul className="list-disc ml-4 mt-1 space-y-1">
+                                    <li>Use the new engine for chart visualizations</li>
+                                    <li>Support for channel types in breakdowns</li>
+                                    <li>Enable conversion goals</li>
+                                    <li>Further improvements in accuracy</li>
+                                    <li>More filters!</li>
+                                </ul>
+                            </div>
+                        </div>
+                    }
+                >
+                    <div
+                        className="flex items-center gap-2 cursor-pointer"
+                        onClick={() => handleToggleEngine(!isUsingNewEngine)}
+                        onMouseEnter={() => setShowPopover(true)}
+                        onMouseLeave={() => setShowPopover(false)}
+                    >
+                        <IconBolt className={isUsingNewEngine ? 'text-warning' : 'text-muted'} />
```

Comment:
This icon is a nice touch, it'd be cool if, when enabled, rather than displaying ths green border for optimized blocks, we:
- displayed the yellow bolt when it was optimized
- displayed the muted bolt when not